subsys/nrf_security/src/psa_crypto_driver_wrappers.c

```diff
@@ -2461,17 +2465,8 @@ psa_status_t psa_driver_wrapper_get_random(psa_driver_random_context_t *context,
 	}
 
 	return PSA_SUCCESS;
-#endif /* defined(PSA_CRYPTO_DRIVER_ALG_PRNG_CC3XX_PLATFORM) */
-
-#if defined(PSA_CRYPTO_DRIVER_ALG_PRNG_OBERON)
-#if defined(PSA_CRYPTO_DRIVER_ALG_CTR_DRBG_OBERON)
-	return oberon_ctr_drbg_get_random(&context->oberon_ctr_drbg_ctx, output, output_size);
-#elif defined(PSA_CRYPTO_DRIVER_ALG_HMAC_DRBG_OBERON)
-	return oberon_hmac_drbg_get_random(&context->oberon_hmac_drbg_ctx, output, output_size);
-#else
-#error "Enable CONFIG_PSA_WANT_ALG_CTR_DRBG or CONFIG_PSA_WANT_ALG_HMAC_DRBG"
+/* TODO: Enforce a DRBG algorithm selection and break the build otherwise. */
```

Comment:
Not if PSA_WANT_GENERATE_RANDOM is disabled. It's a minor use-case, but still a possebility.